src/APIs/quote-def.ts

```diff
@@ -0,0 +1,4 @@
+export interface quoteDef {
+  compliments: Array<string>;
+  motivationalQuotes: Array<Record<string, unknown>>;
```

Comment:
This should be defined differently:

```suggestion
  motivationalQuotes: Array<{
    quote: string;
    author: string;
  }>;
```

we **could** define a `MotivationalQuote` interface, but the "inline" one should be fine for now.